### DISCUSSION TITLE
Coordinates

### DISCUSSION THREAD

**Discussion by queryFF** [Mapsui/Mapsui#2286] [Q&A] [answered]
Very basic question: I'm running the online Blazor samples (Widgets/MouseCoordinatesWidet).  What are the "world" coordinates that are being displayed and how can they be converted to a latitude and longitude that would correspond with Google maps or Bing maps?

**Comment by charlenni** [comment, marked as answer]
I assume, that you see the internal coordinates from Mapsui. 1.0 corresponds to one meter at the equator. North and south it is less then a meter. To convert them to WGS84 use SphericalMercator.ToLonLat(x, y).

**Comment by queryFF** [comment]
That makes so much more sense.  Thank you!  I assume going the other direction would be:
var worldPosition = SphericalMercator.FromLonLat( someLon, someLat );
var screenPosition =  MapControl.Map.Navigator.Viewport.WorldToScreen( worldPosition.X, worldPosition.Y );

There's also a WorldToScreenXY(...) but it seems to give the same results so the difference between them isn't clear.

**Reply by pauldendulk** [reply]
WorldToScreenXY is the same but returns a Tuple. It is like an overload of WorldToScreen but since the difference is in the return value and not in the parameters it can not be treated like a regular overload.
